src/navigate/Navigator.js

```diff
@@ -111,9 +111,7 @@ define([
             // Compute the far clip distance based on the current eye altitude. This must be done after computing the
             // modelview matrix and before computing the near clip distance. The far clip distance depends on the
             // modelview matrix, and the near clip distance depends on the far clip distance.
-            this.farDistance = WWMath.horizonDistanceForGlobeRadius(globeRadius, eyePos.altitude);
-            if (this.farDistance < 1e3)
-                this.farDistance = 1e3;
+            this.farDistance = WWMath.horizonDistanceForGlobeRadius(globeRadius, eyePos.altitude) + 1.5e6;
```

Comment:
If you have time to make the necessary changes, we can approve this PR. Otherwise, I'd be happy to open a new PR from an in-repository branch.